src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -596,15 +577,54 @@ void GcsPlacementGroupScheduler::AcquireBundleResources(
   }
 }
 
+absl::flat_hash_map<scheduling::NodeID, ResourceRequest> ToNodeBundleResourcesMap(
```

Comment:
NOTE: This is a help function to convert `BundleLocations` to per-node resource request.